test/e2e_node/mount_rro_linux_test.go

```diff
@@ -135,6 +136,7 @@ var _ = SIGDescribe("Mount recursive read-only [LinuxOnly]", framework.WithSeria
 		}) // Context
 		ginkgo.Context("when the runtime does not support recursive read-only mounts", func() {
 			f.It("should accept non-recursive read-only mounts", func(ctx context.Context) {
+				e2eskipper.SkipUnlessFeatureGateEnabled(features.RecursiveReadOnlyMounts)
```

Comment:
(Looks like we can't use this. The test is now always skipped, even with `TEST_ARGS='--service-feature-gates=RecursiveReadOnlyMounts=true --kubelet-flags="--feature-gates=RecursiveReadOnlyMounts=true"`)

Reply:
How about just adding one skipper to the SIGDescribe container? for example,

```go
ginkgo.BeforeEach(func(ctx context.Context) {
    e2eskipper.Skipf("feature gate RecursiveReadOnlyMounts is not enabled")
})
```

BTW, which job will run these tests?

Reply:
Sorry, my comment was wrong, please ignore